cms/src/components/groups/GroupEditor.vue

```diff
@@ -263,6 +263,19 @@ const discardChanges = () => {
     newGroupName.value = props.group.name;
 };
 
+const duplicateGroup = async () => {
+    const duplicatedGroup = { ...toRaw(props.group) };
+    duplicatedGroup._id = "";
+    duplicatedGroup.name = `Copy of ${duplicatedGroup.name}`;
+    await createGroup(duplicatedGroup);
+
+    addNotification({
+        title: `Group "${duplicatedGroup.name}" duplicated successfully`,
```

Comment:
I think this should either say the old name (`Group "Editors Private" duplicated successfully`) or say "created" instead of "duplicated" (`Group "Copy of Editors Private" created successfully`)